### DISCUSSION TITLE
DAI 2023-2024 - Practical work 1 - Thomas Vuilleumier and Sebastian Diaz

### DISCUSSION THREAD

**Discussion by Cobora2001** [heig-vd-dai-course/heig-vd-dai-course#68] [Show and tell]
[Our Repository](https://github.com/MolinaElGringo/DAI_Pratical_Work1)

[Our Release](https://github.com/MolinaElGringo/DAI_Pratical_Work1/releases/tag/completed)

Our latest hash: b15090e

**Comment by ludelafo** [comment]
Could you please tell us a bit about what you plan to do for this practical work? :)

**Reply by Cobora2001** [reply]
Hi,

As a first step at least, we will try to implement a CTR-f, meaning copying the content of a file in another (or the same) with highlighted letters or groups of letters (meaning that we put characters defined by the user before and after our pattern, making it visible).

We will come back to you if we have more questions.

Edit : We have also made a UpperCase and LowerCase, to try having subcommands

**Reply by Cobora2001** [reply]
Description has been updated with the commit hash and the release we created.
